code/game/machinery/porta_turret/portable_turret_cover.dm

```diff
@@ -82,6 +82,7 @@
 	parent_turret.attack_animal(user)
 
 /obj/machinery/porta_turret_cover/attack_hulk(mob/living/carbon/human/user)
+	SHOULD_CALL_PARENT(0)
```

Comment:
If you're not willing to go through the whole proc chain to make them all call parent don't bother adding the linting yet.